.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+**Description**:
+- ...
+
+**Checklist**:
+- [ ] My PR is ready for code review
+- [ ] I have added some tests, if applicable
+- [ ] My code respects the [PEP8 style guide](https://www.python.org/dev/peps/pep-0008/)
```

Comment:
I don't know if that is necessary. I think that for now we have no guaranty that our current code base is PEP8 compliant. The only thing we know is that the codebase is compliant with the Google-like linting rules that we have in the `pylintrc`

Reply:
+ I find it very hard to know if your code is PEP8 compliant from juste the guidelines provided in the URL. The linting test has the advantage to tell you precisely what's wrong

Reply:
Indeed